src/perfect-number.js

```diff
@@ -0,0 +1,12 @@
+function isPerfect(n) {
+  let sum = 0;
+  for (let i = 1; i <= n / 2; i += 1) {
+    if (n % i === 0) sum += i;
```

Comment:
n % !==0 then return false

Reply:
no, it needs to find all divisors and add them.